docs/usage.md

```diff
@@ -42,23 +42,12 @@ The Rust library crate provides the following capabilities:
 * `no_interleaved_io` forces fully-synchronous I/O; otherwise, the library uses threaded I/O for some operations to improve performance.
 * `fetch_remote_manifests` enables the verification step to retrieve externally referenced manifest stores.  External manifests are only fetched if there is no embedded manifest store and no locally adjacent .c2pa manifest store file of the same name.
 * `json_schema` is used by `make schema` to produce a JSON schema document that represents the `ManifestStore` data structures.
-* `openssl_ffi_mutex` prevents multiple threads from accessing the C OpenSSL library simultaneously. (This library is not re-entrant.) In a multi-threaded process (such as Cargo's test runner), this can lead to unpredictable behavior.
 
 ### New API
 
-### Enabling 
+The new api is now enabled by default, the `unstable_api` feature is no longer used.
```

Comment:
Please capitalize API consistently.

Reply:
Also, this is a comma splice. S/be:

The new API is now enabled by default. The `unstable_api` feature is no longer used.